### ISSUE TITLE
RequestBackground created malformed `.desktop` files.

### ISSUE THREAD

**Issue by midnight-wonderer** [flatpak/xdg-desktop-portal#1481]
### Operating System

Ubuntu 22.04

### XDG Desktop Portal version

Other

### XDG Desktop Portal version (Other)

I don't know where to find the version. I just use the latest available stable Flatpak.

### Desktop Environment

GNOME

### Expected Behavior

When `RequestBackground` is called with `autostart: TRUE`, it should produce a valid `.desktop` entry for the app.

### Current Behavior

A malformed `.desktop` is produced.

### Steps to Reproduce

I am really just an end user. I don't know much details.
Could you check this out? https://github.com/keepassxreboot/keepassxc/issues/11379
Basically, they are calling "org.freedesktop.portal.Background", "RequestBackground" for the autostart. But a malformed `.desktop` was produced as a result.

### Anything else we should know?

I hope I know what else you need to know.

**Comment by swick** [comment]
This is a rather old version. Is this reproducible on current versions as well?

**Comment by midnight-wonderer** [comment]
I am willing to try anything to aid in bug discovery in my environment.
However, the limitation is that I don't have the same context as one would expect from a developer using Flatpak. So, a little bit verbosity might help.

As for the version, I believe I have the latest stable available on my platform.
~~~
$ flatpak --version
Flatpak 1.14.10
~~~

~~~
$ uname -a
Linux PC 6.8.0-47-generic #47~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Oct  2 16:16:55 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
~~~

~~~
$ flatpak list | grep -i platform
Name                                                Application ID                                                    Version                                Branch                    Installation
Freedesktop Platform                                org.freedesktop.Platform                                          freedesktop-sdk-23.08.23               23.08                     system
Freedesktop Platform                                org.freedesktop.Platform                                          freedesktop-sdk-24.08.4                24.08                     system
Mesa                                                org.freedesktop.Platform.GL.default                               24.2.3                                 23.08                     system
Mesa (Extra)                                        org.freedesktop.Platform.GL.default                               24.2.3                                 23.08-extra               system
Mesa                                                org.freedesktop.Platform.GL.default                               24.2.5                                 24.08                     system
Mesa (Extra)                                        org.freedesktop.Platform.GL.default                               24.2.5                                 24.08extra                system
Intel                                               org.freedesktop.Platform.VAAPI.Intel                                                                     23.08                     system
Intel VAAPI driver                                  org.freedesktop.Platform.VAAPI.Intel                                                                     24.08                     system
ffmpeg-full                                         org.freedesktop.Platform.ffmpeg-full                                                                     23.08                     system
openh264                                            org.freedesktop.Platform.openh264                                 2.1.0                                  2.2.0                     system
openh264                                            org.freedesktop.Platform.openh264                                 2.4.1                                  2.4.1                     system
GNOME Application Platform version 45               org.gnome.Platform                                                                                       45                        system
GNOME Application Platform version 46               org.gnome.Platform                                                                                       46                        system
GNOME Application Platform version 47               org.gnome.Platform                                                                                       47                        system
KDE Application Platform                            org.kde.Platform                                                                                         5.15-23.08                system
KDE Application Platform                            org.kde.Platform                                                                                         6.6                       system
QGnomePlatform                                      org.kde.PlatformTheme.QGnomePlatform                                                                     5.15-23.08                system
QGnomePlatform                                      org.kde.PlatformTheme.QGnomePlatform                                                                     6.6                       system
QGnomePlatform-decoration                           org.kde.WaylandDecoration.QGnomePlatform-decoration                                                      5.15-23.08                system
~~~

The package in question is
~~~
KeePassXC                                           org.keepassxc.KeePassXC                                           2.7.9                                  stable                    user
~~~

**Comment by bbhtt** [comment]
This was fixed in https://github.com/flatpak/xdg-desktop-portal/commit/ce57c6c72163d069d5b418d64bd0e53445dd8d44, you have to ask Ubuntu to pick that patch. They are on a pretty old version 1.14.3.
